app/javascript/bundles/Identity/components/KnownIdentity.jsx

```diff
@@ -9,16 +9,14 @@ export default class KnownIdentity extends React.Component {
   render() {
     const {
       id,
-      last_name,
+      family_name,
       first_name,
-      middle_name = "",
+      second_name = "",
       third_name = "",
-      date_of_birth,
-      country_of_birth,
+      country,
```

Comment:
which country is this? The country the identity was issued by, or the country of birth?